java/code/src/com/redhat/rhn/taskomatic/task/repomd/DebRepositoryWriter.java

```diff
@@ -124,6 +124,11 @@ public void writeRepomdFiles(Channel channel) {
                 for (PackageDto pkgDto : packageBatch) {
                     writer.addPackage(pkgDto);
                 }
+                log.info("Processed {} packages", i + packageBatch.getEnd());
+                if (commitTransaction) {
+                    // commit pre generated package snippets in the cache
+                    HibernateFactory.commitTransaction();
```

Comment:
Do we really want to commit for each package?

Reply:
This is a batch. We commit every 1000 packages